tests/posix/threads_base/src/pthread.c

```diff
@@ -605,22 +602,10 @@ static void *test_pthread_setschedprio_fn(void *arg)
 	return NULL;
 }
 
-ZTEST(pthread, test_pthread_setschedprio)
+ZTEST(posix_threads_base, test_pthread_setschedprio)
```

Comment:
This test belongs in the `xsi_realtime_threads` testsuite with the `_POSIX_THREAD_PRIORITY_SCHEDULING` Option.